lib/web_ui/lib/src/engine/window.dart

```diff
@@ -281,14 +281,14 @@ class EngineWindow extends ui.Window {
   static List<ui.Locale> parseBrowserLanguages() {
     // TODO(yjbanov): find a solution for IE
     final bool languagesFeatureMissing = !js_util.hasProperty(html.window.navigator, 'languages');
-    if (languagesFeatureMissing || html.window.navigator.languages.isEmpty) {
+    if (languagesFeatureMissing || html.window.navigator.languages!.isEmpty) {
```

Comment:
Sounds like we don't need the `js_util.hasProperty` hack on the previous line. We can just null-check `window.navigator.languages` and remove further null checks.

Reply:
Yup, we can simplify this. Done.